cmd/quickstart.go

```diff
@@ -258,6 +267,20 @@ func quickstartExec(ctx context.Context, flags QuickstartFlags) error {
 		quickstartObj.WorkflowFile.Sources[sourceName].Inputs[0].Location = workflow.LocationString(referencePath)
 	}
 
+	// If we are using a blueprint template, the original location will be a
+	// tempfile. We want therefore to move the tempfile to the output directory,
+	// and update the workflow file to point to the new location.
+	if quickstartObj.IsUsingBlueprint {
```

Comment:
Why don't we do this as the first thing? If you're using the blue print -> download it into openapi.yaml then do everything else? Though maybe I'm not understanding the code fully yet.

Reply:
Because the outdir hasn't been created until this point